### DISCUSSION TITLE
an unexpected keyword argument while use unet_segmentation_3d_catalyst.ipynb

### DISCUSSION THREAD

**Discussion by liyh1999** [Project-MONAI/MONAI#6053] [Q&A] [answered]
hello,I'm following a 3d segmentation tutorial：unet_segmentation_3d_catalyst.ipynb，but I get an error：TypeError: __init__() got an unexpected keyword argument 'input_target_key'
This appears in 
“runner = mRunner(
    input_key="img", input_target_key="seg", output_key="logits"
) ”
mRunner is:
"class mRunner(catalyst.dl.SupervisedRunner):
    def forward(self, batch):
        if self.is_train_loader:
            output = {self.output_key: self.model(batch[self.input_key])}
        elif self.is_valid_loader:
            roi_size = (96, 96, 96)
            sw_batch_size = 4
            output = {
                self.output_key: sliding_window_inference(batch[self.input_key], roi_size, sw_batch_size, self.model)
            }
        elif self.is_infer_loader:
            roi_size = (96, 96, 96)
            sw_batch_size = 4
            batch = self._batch2device(batch, self.device)
            output = {
                self.output_key: sliding_window_inference(batch[self.input_key], roi_size, sw_batch_size, self.model)
            }
            output = {**output, **batch}
        return output"
there doesn't seem to be  input_target_key？anyone can help me ？

**Comment by KumoLiu** [comment, marked as answer]
Hi @liyh1999, did you change anything in the tutorial? May I know your MONAI version and catalyst version? And could you please share the whole error message? Thanks!

**Reply by liyh1999** [reply]
I did not make changes,Below is my config
MONAI version: 1.0.1
Numpy version: 1.23.4
Pytorch version: 1.13.0
MONAI flags: HAS_EXT = False, USE_COMPILED = False, USE_META_DICT = False
MONAI rev id: 8271a193229fe4437026185e218d5b06f7c8ce69
MONAI __file__: D:\anconda3\envs\autuseg\lib\site-packages\monai\__init__.py

Optional dependencies:
Pytorch Ignite version: NOT INSTALLED or UNKNOWN VERSION.
Nibabel version: 4.0.2
scikit-image version: 0.19.3
Pillow version: 9.2.0
Tensorboard version: 2.11.0
gdown version: NOT INSTALLED or UNKNOWN VERSION.
TorchVision version: 0.14.0
tqdm version: 4.64.1
lmdb version: NOT INSTALLED or UNKNOWN VERSION.
psutil version: 5.9.0
pandas version: 1.5.2
einops version: NOT INSTALLED or UNKNOWN VERSION.
transformers version: NOT INSTALLED or UNKNOWN VERSION.
mlflow version: NOT INSTALLED or UNKNOWN VERSION.
pynrrd version: NOT INSTALLED or UNKNOWN VERSION.

**Reply by KumoLiu** [reply]
Hi @liyh1999, I think it may be a different version caused by `catalyst`, in the tutorial, it used `catalyst==20.07`. Thanks!

**Reply by liyh1999** [reply]
thanks,The problem was successfully resolved
